### ISSUE TITLE
Spring Attack, Dive Attack, etc. has too much value for 1 Stamina

### ISSUE THREAD

**Issue by Marcloure** [mlenser/kryx-rpg-issues#948]
I have an issue with these maneuvers that allow you to make a Dash and an attack. So, they greatly reduce the value push and movement penalty (specially push). See: you push the target 2 or 3 meters and move 5 meters away so it can't reach you with its normal speed. But then it uses Spring Attack to Dash and catches up with you and still makes an attack.

For a time I thought that was fine, since you are forcing the target to use the Spring Attack maneuver instead of any other maneuver. However, the creature is only expending 1 stamina to make the dash, they will still make a second attack with another maneuver (assuming it has a limit higher than 1) afterwards.

Then I thought: pushing someone out of reach is like reducing their stamina limit by 1, since they will need to spend 1 stamina do use this maneuver. However, that is also not true, since they still get 1d8 damage from this maneuver, and expending this 1 stamina to enhance any other maneuver would in many cases just increase the damage by d8. The lost value of Spring Attack is in many cases non-existent, since this 1 stamina wouldn't bring a benefit if used in another maneuver.

Also, there is no reason to enhance these maneuvers above 1 Stamina.

---

I think the spreadsheet considers dash being much less valuable than it really is. If a creature is using a maneuver to dash and attack, that means the maneuver is what is allowing the attack to happen in the first place. It's like the dash deals the damage of an attack, does that make sense?

So, my most simplistic suggestion is to remove the initial extra d8 damage from spring attack and the likes (and rebalance dashing spells as well). This way, using 1 Stamina on Spring Attack is a cost when compared to enhancing another maneuver.

**tl;dr:** Spring attack allows an "extra attack" for 1 Stamina, which is already very valuable. There is no need for the extra d8 on top of it.

**Comment by Marcloure** [comment]
On the same note, Spring Attack without the d8 has a similar value of Retaliate:

Retaliate allows you to make an attack when you usually can not.
Spring Attack allows you to make one attack when you usually can not.

**Comment by itamarcu** [comment]
Also note:  when pushing (e.g. with Knockback), you deal +1d8 damage and push the enemy only if the enemy fails a Reflex save (~55%). However, when dashing (e.g. with Spring Attack), you deal +1d8 damage and reach the enemy always, without needing to roll.  In such a "trade", each creature spends 1 stamina and gains bonus damage, but only one of them needs to roll for the additional effect.

Pushing still has extra potential utility for moving creatures into bad spots (like into traps or next to martial friends), which gives more value than just forcing the enemy creature to move X extra feet towards you.  But I don't know if a maneuver can be intentionally weaker due to the fact it can be used more efficiently.  Plus, there's some utility in dashing too.

**Comment by Marcloure** [comment]
Yeah, but my main concern is that:

- Spring Attack + Another Maneuver (2 attacks + 2d8)

In many cases deal the same damage as

- Another Maneuver Enhanced + attack (2 attacks + 2d8)

I think that shouldn't be true. There should be some penalty when you try to reach someone farther away.

**Comment by itamarcu** [comment]
> There should be some penalty when you try to reach someone farther away

Also consider [Lunge](https://www.kryxrpg.com/maneuvers/lunge) which gives +1d8 and increases your melee reach by just 1 meter, and [Distant Shot](https://www.kryxrpg.com/maneuvers/distant-shot) which gives +1d8 and doubles your ranged attack which already had range in the first place (usually turning a disadvantage attack into a normal attack)

**Comment by mlenser** [comment]
> I have an issue with these maneuvers that allow you to make a Dash and an attack. So, they greatly reduce the value push and movement penalty (specially push). See: you push the target 2 or 3 meters and move 5 meters away so it can't reach you with its normal speed. But then it uses Spring Attack to Dash and catches up with you and still makes an attack.

100% intended. These maneuvers and spells are intended to get creatures into the battle at a cost (of resources).

> Then I thought: pushing someone out of reach is like reducing their stamina limit by 1, since they will need to spend 1 stamina do use this maneuver. However, that is also not true, since they still get 1d8 damage from this maneuver

They are using this maneuver instead of another. Other maneuvers over utility/conditions which Sping Attack and similar maneuvers/spells do not (beyond the movement).

> Also, there is no reason to enhance these maneuvers above 1 Stamina.

Sure there is. Maybe you only have 1 action left or maybe you dash and then use Spring Attack so only have 1 action.

> So, my most simplistic suggestion is to remove the initial extra d8 damage from spring attack and the likes (and rebalance dashing spells as well). This way, using 1 Stamina on Spring Attack is a cost when compared to enhancing another maneuver.

Using Spring Attack and similar maneuvers is already a cost.

**Comment by mlenser** [comment]
> Also note: when pushing (e.g. with Knockback), you deal +1d8 damage and push the enemy only if the enemy fails a Reflex save (~55%). However, when dashing (e.g. with Spring Attack), you deal +1d8 damage and reach the enemy always, without needing to roll. In such a "trade", each creature spends 1 stamina and gains bonus damage, but only one of them needs to roll for the additional effect.

Some maneuvers have a roll and some don't. Lunge, Distant Shot, Fan of Knives, etc.

> There should be some penalty when you try to reach someone farther away.

There is. It costs resources which could be used on other maneuvers that have effects.

**Comment by Marcloure** [comment]
> These maneuvers and spells are intended to get creatures into the battle at a cost (of resources).

I know it's intended, that's why I don't think the dash should be changed. I just think the maneuver is valuable enough with just that, there is no need for the extra initial d8. Gaining an extra action is pretty strong by itself.

Also, these maneuvers have a 0.9 multiple because you need to move at least 4 meters, but obviously you are not using them if you can reach your target normally, so that isn't really a downside, no?

**Comment by mlenser** [comment]
I've been mentally pondering this for a while now. I'm not sure this case is so different from a spell with range for a martial gish or a maneuver that can be used at range. Both cases would also be used when you can't get in melee. Range on the spells is accounted for, but so is the dash.

We also have cases like Rising Phoenix that allow you to fly before attacking.

Some calculations without the dash/fly:
- Rising Phoenix has ~14 damage value. Without the fly, it has ~11.4
- Spring Attack has ~6.5 damage value. Without the dash, it has ~4.4
- Dive Attack has ~7.4 damage value. Without the dash, it has ~5.3

Without the dash, they're essentially just damage boosts. The main functionality is the dash.

Using Spring Attack comes at the cost of not using a better maneuver now or later. If you Spring Attack to get in range, you have 1 less stamina to spend this round and until your next short rest.

hm...

**Comment by Marcloure** [comment]
> 1 less stamina to spend this round and until your next short rest.

Specially for NPCs, which all have spring attack, this second part hardly matters. Also, the first part only matters if you would use this 1 stamina to do more than increase damage by 1d8.

Just to give context, I made this issue after i run a wyvern with some ogres. I noticed that spamming Dive Attack + Inject Venom is much better than just use Inject Venom at 2 stamina, and when the druid pushed the ogres with avalanche, they just spring attack back into position at the cost of 1 stamina. As they still gained extra damage, the spell was much less effective than desirable.

**Comment by mlenser** [comment]
> Specially for NPCs, which all have spring attack, this second part hardly matters. Also, the first part only matters if you would use this 1 stamina to do more than increase damage by 1d8.

Both matter for most creatures. Some examples:
- [Aarakocra](https://www.kryxrpg.com/creatures/aarakocra) (challenge ¼) only has 1 stamina so using it on dive attack is costly.
- [Adult Kruthik](https://www.kryxrpg.com/creatures/adult-kruthik) (challenge 2) has 5 stamina and a limit of 1. Using Spring attack means it can't use another maneuver like Ambush (stealth & damage), Lead the Attack (adv), Pounce (proning the enemy), or Tear Flesh (bleeding).
- [Giant Ape](https://www.kryxrpg.com/creatures/giant-ape) (challenge 7) has 14 stamina and a limit of 3. Using Spring attack means it can use 1 less stamina on other maneuvers or can only use 1 other maneuver this turn when it may have been able to use two maneuvers. For example, it could've used disorient twice against two creatures or it could've used hammer spin and then knockback. None of those are an option if it uses Spring Attack.

> i run a wyvern with some ogres. I noticed that spamming Dive Attack + Inject Venom is much better than just use Inject Venom at 2 stamina

There is an issue with Inject Venom in that it is a weak maneuver. Below you can see that it is about 65-80% as effective as the average maneuver. For this reason and others I've thought of removing it.

![image](https://user-images.githubusercontent.com/235857/132504128-95966331-6d29-416e-8668-57787dc540b8.png)

But also, Dive Attack is a great maneuver. Even if the value of it was increased it wouldn't make it used less as it can allow a creature to dive into combat and out with ease.

> when the druid pushed the ogres with avalanche, they just spring attack back into position at the cost of 1 stamina. As they still gained extra damage, the spell was much less effective than desirable.

That has a cost for those ogres. They are very similar to the Adult Kruthik in that they have 5 stamina and a limit of 1. Meaning if they Spring Attack then they can't do all kinds of other great things like Disorient. So the push was effective at stopping potentially very bad things from happening.

**Comment by mlenser** [comment]
FYI: I reworked Inject Venom so it is more valuable.

**Comment by Marcloure** [comment]
> Meaning if they Spring Attack then they can't do all kinds of other great things like Disorient.

I understand this, but I am not sure losing "On a hit, the creature must succeed on a Fortitude saving throw or it can’t take reactions for the duration" is enough to make pushing creatures away or opening distance worth it. I think losing the initial extra d8 damage should also happen, frankly I think moving 5 meters (often more) is quite worth 1 Stamina.

I don't have much else to say. If you think this is intended, then it's fine.

**Comment by mlenser** [comment]
> I am not sure losing "On a hit, the creature must succeed on a Fortitude saving throw or it can’t take reactions for the duration" is enough to make pushing creatures away or opening distance worth it.

Sure, then look at the other maneuvers that are more valuable at 1 stamina die:
- Debilitating crush can cause a creature to deal half damage. That can be quite impactful
- Hammer Spin can do ~4.5 damage to ~2-3 creatures. So using spring attack means they can't do ~9-13.5 damage
- Hammering Blow effectively does 2 extra damage, which is not used.
- Hammer Through, like Hammer Spin means they can't do ~9-13.5 damage
- Knockback means they may be pushed away, which could be into bad terrain or could cause the enemy to have to dash to get back into melee (if the Ogre moves away).

Some of these would be great and losing out on them is costly.

_______________

> I think losing the initial extra d8 damage should also happen, frankly I think moving 5 meters (often more) is quite worth 1 Stamina.

You've solely focused on Spring Attack. Any change would have to be done holistically. Meaning it would apply to Dive Attack, Spring Attack, Furious Charge, Shadow Jaunt / Teleport, Rising Phoenix, Telekinetic Charge, and Zephyr Strike.
Changing what you're suggesting would make all of these options much less powerful. It would make them only useful when you're out of range and need to get into melee. Shadow Jaunt, for example, would then be a poor choice to jump into and out of shadows as it would deal less damage than other comparable options.

What you're suggesting wouldn't just impact Spring Attack. It would impact many other options in negative ways which is not desireable.

**Comment by mlenser** [comment]
Lastly, melee characters are meant to be able to get into melee. Spring Attack has a cost (in stamina and not using another maneuver), but it isn't meant to be a super high cost. Many creatures don't have ranged options and this is a design choice to allow such creatures to engage in melee without being easily kited or having other difficulties. They can still be kited and will have issues with difficult terrain, but it is purposeful that melee creatures have an easier time engaging than RAW.

So based on that design choice, this seems to be working as intended.
